src/components/FeaturePanel/PublicTransport/route/Halt.tsx

```diff
@@ -0,0 +1,46 @@
+import TripOriginIcon from '@mui/icons-material/TripOrigin';
+
+export type HaltSection = 'start' | 'middle' | 'end';
+
+type Props = {
+  color: string;
+  isFirst: boolean;
+  isLast: boolean;
+  hold?: boolean;
+};
+
+export const Halt = ({ color, isFirst, isLast, hold = true }: Props) => (
+  <div
+    style={{
+      width: '1rem',
+      display: 'flex',
+      flexDirection: 'column',
+      justifyContent: 'center',
+      alignItems: 'center',
+    }}
+  >
+    <div
+      style={{
+        width: '4px',
+        height: '100%',
+        background: isFirst ? 'transparent' : color,
+      }}
+    />
+    {hold && (
```

Comment:
I don't understand what does this prop mean, can you please explain and then we can perhaps find a more suitable name?

Reply:
At all stations a train **hold**s so they get a circle on the left line. The *show more*/*show less* buttons should not get that circle as a train doesn't **hold** there.
`hold` shows/hides that circle:
![image](https://github.com/user-attachments/assets/2cb497f2-6516-4e85-8c8f-39f3e0072eca)

Reply:
see new comment [below](https://github.com/zbycz/osmapp/pull/626#discussion_r1790523648)